### PR TITLE
Keep users selection persistent even after server-side pagination

### PR DESCRIPTION
**Pull request by jonkiky**
## Description

Keep users selection persistent even after server-side pagination

### CODE REVIEW

**Comment by jonkiky** [comment]
waiting for bento-compents get updated.

File: src/components/serverPaginatedTable/serverPaginatedTable.js [hunk 1]
```diff
@@ -40,9 +39,14 @@ class ServerPaginatedTableView extends React.Component {
 
   getSrcData = () => this.props.data;
 
+  rowsSelectedTrigger = (displayedData) => this.props.options.rowsSelectedTrigger(
```
Comment (AjayDoddapaneni) [inline src/components/serverPaginatedTable/serverPaginatedTable.js:42]: Can you add an edge case where we don't need rowsselectedTrigger(Which will benefit the cart)

File: src/pages/dashboardTab/components/tabController.js [hunk 2]
```diff
@@ -224,7 +224,7 @@ const tabController = (classes) => {
   const TABContainers = tabContainers.map((container) => (
     <TabContainer id={container.id}>
       <TabView
-        options={getOptions(container, classes)}
+        options={getOptions(container, classes, undefined, undefined)}
```
Comment (AjayDoddapaneni) [inline src/pages/dashboardTab/components/tabController.js:227]: Can we add the default values in bento-tools for this function. So we don't need to add undefined in here

**Comment by skiran86** [comment]
@jonkiky Conflicts

**Comment by skiran86** [comment]
@jonkiky Please make sure you save the changes from https://github.com/CBIIT/bento-frontend/pull/222/files#diff-db9012241117b993afadb02900a721f791e96bba80e852c64fa3b17d7939eae5

**Comment by AjayDoddapaneni** [comment]
Why don't we don't read the custom checkboxes util functions for now. As this might change project by project basis?

**Comment by jonkiky** [comment]
> Why don't we don't read the customcheckboxes function from util functions for now. As this might change project by project basis?

Not sure about this, but I am to keep it in the util .

**Comment by AjayDoddapaneni** [comment]
Fine with me then

**Review by AjayDoddapaneni — APPROVED** [review]

**Pull request merged**